### ISSUE TITLE
Example 1 and Example 3 solutions in the doc differ from my solution

### ISSUE THREAD

**Issue by krashish8** [VROOM-Project/vroom#672]
Are there any specific parameters used in the routing engine to generate the solution for Example 1 and Example 3?

I tried generating the solution using the OSRM demo server (http://router.project-osrm.org) and the VROOM demo server (http://solver.vroom-project.org) using VROOM v1.11.0, but both the results differed from the one in the documentation.

### To reproduce:

**Example 1:**

Using OSRM demo server: [Diff of OSRM soln vs doc soln](https://gist.github.com/krashish8/72339dac8e1009d7e0f776533674f533#file-example1-osrmdemo-vs-doc-diff) - The job with ID 6 got unassigned when solved using the OSRM demo server
```
bin/vroom -i docs/example_1.json -a car:router.project-osrm.org -p car:80 -g | jq
```

Using Vroom demo server: [Diff of Vroom soln vs doc soln](https://gist.github.com/krashish8/72339dac8e1009d7e0f776533674f533#file-example1-vroomdemo-vs-doc-diff) - Pretty close solution, the numbers differ a bit
```
curl --header "Content-Type:application/json" --data "`cat docs/example_1.json`" http://solver.vroom-project.org | jq
```

**Example 3:**

Using OSRM demo server: [Diff of OSRM soln vs doc soln](https://gist.github.com/krashish8/72339dac8e1009d7e0f776533674f533#file-example3-osrmdemo-vs-doc-diff) - Got an extra delay violation when solved using OSRM demo server
```
bin/vroom -c -i docs/example_3.json -a car:router.project-osrm.org -p car:80 -g | jq
```

Could not try Example 3 with the Vroom demo server, because the plan mode URL is unknown to me.

Any possible reasons why the results differ significantly, and how can I generate a similar solution as demonstrated by the documentation (maybe the parameters of the routing server differ?)?

**Comment by jcoupey** [comment]
TL;DR there is no real hope to reproduce anything without actual access to the same routing stack. You may be interested in [the wiki page on standalone instances](https://github.com/VROOM-Project/vroom/wiki/Standalone-problem-instance).

> Any possible reasons why the results differ significantly

Plenty, the results can differ based on any combination of the following:

- routing engine of choice and version;
- routing profile (and any change applied to it);
- underlying OSM data.

So even if you're comparing OSRM with the `car` profile, I'm not surprised you're getting different solutions. The OSM data and OSRM version are pretty much outdated on the vroom demo server, and the OSRM demo server hosted by FOSSGIS does not use exactly the default profile (I think). The actual output versioned in the repo was obtained by myself on another private OSRM instance.

The reported solutions for the examples mostly serve as a way to showcase the various keys found in input/output, which is by the way why deprecated keys are omitted.

**Comment by krashish8** [comment]
Oh, thanks a lot!
That clarifies my doubt.
